### PR TITLE
Update rules_catalog.md

### PR DESCRIPTION
**Pull request by Men-hau**
line 167 adding Similar strategy field on custom rule creation  (new feature available)

### CODE REVIEW

**Comment by ka0ula** [comment]
Why did you use this format "RULE.form.alert-properties.similarity-strategy" instead of just giving the name of the label? "Similarity strategy"

**Comment by Men-hau** [comment]
This is currently display as follow

image

Maybe, we need to change it in SEKOIA.IO before putting the label you propose

**Comment by ka0ula** [comment]
@Men-hau the label is already "Similarity strategy". Try to refresh your browser or delete the cache ;)

File: docs/xdr/features/detect/rules_catalog.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ For Custom rule, you will be able to edit its main definition:
 
 - General definition of the rule
 - Detection Pattern
-- Security alerts
+- Security alerts  (event field can be selected to define the similarity strategy in the box `RULE.form.alert-properties.similarity-strategy`)
```
Comment (ka0ula) [inline docs/xdr/features/detect/rules_catalog.md:167]:
```suggestion
- Security alerts (event fields can be selected to define the similarity strategy in the section `Similarity strategy`). 
```

**Review by ka0ula — APPROVED** [review]

**Pull request merged**